syntax = "proto3";

package fintekkers.models.util;

message LocalDateProto {
  uint32 year = 1;
  uint32 month = 2;
  uint32 day = 3;
}